pytest_logger/plugin.py

```diff
@@ -13,6 +14,28 @@
 else:
     string_type = str
 
+os_symlink = getattr(os, "symlink", None)
+if callable(os_symlink) is False and platform.system() == 'Windows':
+    
+    def symlink_win(source, linkname):
+        """ Symlink implementation on Windows platform, working from Vista
+        inspired by http://stackoverflow.com/questions/6260149/os-symlink-support-in-windows
+        """
+        import ctypes
+        flags = 1 if source is not None and os.path.isdir(source) else 0
+        try:
+            linkname = linkname.replace('/', '\\')
+            csl = ctypes.windll.kernel32.CreateSymbolicLinkW
```

Comment:
I don't understand why python3 version was not backported to pyton2.7. function CreateSymbolicLinkW uses wide characters, this could be an issue. I found another propsed function CreateSymbolicLinkA but don't work.

Anyway with #5 this could be obsoleted, it is not necessary to use symlinks if user can define out dir in any place and see result and also backup it. And symlink can be then created manualy.